examples/example-app/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.4.0")
```

Comment:
Wow didn't know of that plugin... we struggled with local multi-JVM runs in the project for a while 😯